### PR TITLE
Integrations, take 2

### PR DESCRIPTION
**Pull request by michaelklishin**
This is #11 by @Licenser without branches merged into it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* [![Build Status](https://travis-ci.com/Kyorai/cuttlefish.svg?branch=fix-tests)](https://travis-ci.com/Kyorai/cuttlefish)
```
Comment (dumbbell) [inline README.md]: One last thing: the branch is incorrect. It should be `develop`.

**Review by dumbbell — CHANGES_REQUESTED** [review]

**Comment by michaelklishin** [comment]
@dumbbell done.

**Review by dumbbell — APPROVED** [review]

**Comment by dumbbell** [comment]
Crap I just noticed that for this project, the URL is on travis-ci.org, not travis-ci.com (unlike the RabbitMQ fork...):
https://travis-ci.org/Kyorai/cuttlefish.svg?branch=develop

No idea what the difference might be, but the image is unavailable on the .com URL...

**Comment by dumbbell** [comment]
Likewise for the project URL on Travis CI (s/com/org/):
https://travis-ci.org/Kyorai/cuttlefish

**Comment by michaelklishin** [comment]
@dumbbell done.

**Comment by dumbbell** [comment]
Thanks!

**Pull request merged**